src/addons/jest-matchers.js

```diff
@@ -18,6 +18,11 @@ expect.extend({
   },
 
   toHaveStatus (action, status) {
+    if (action instanceof Action === false) {
+      throw new TypeError("toHaveStatus expects an Action. Received " +
+                          (action ? "a " + action.constructor.name : action) + ".")
```

Comment:
I want to account for other values, like `undefined`, a string, or a number, or something else. This at least calls `toString` on it.